### PR TITLE
chore: Change "pool" to "pair" in the text description of liquidity

### PR DESCRIPTION
**Pull request by Woody810**

### CODE REVIEW

File: packages/localization/src/config/translations.json [hunk 1]
```diff
@@ -1788,7 +1788,7 @@
   "Cannot estimate zap amount": "Cannot estimate zap amount",
   "Zap in amount too low": "Zap in amount too low",
   "Insufficient %token% balance": "Insufficient %token% balance",
-  "Your pool share": "Your pool share",
+  "Your pair share": "Your pair share",
```
Comment (5cf2a7d4bf6e4cdb64b37b7a03b9f2f7) [inline packages/localization/src/config/translations.json:1791]: Your share in trading pair

File: apps/aptos/components/Liquidity/components/AddLiquidityButton.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export default function AddLiquidityButton({
   const [onPresentAddLiquidityModal] = useModal(
     <ConfirmAddLiquidityModal
       parsedAmounts={parsedAmounts}
-      title={noLiquidity ? t('You are creating a pool') : t('You will receive')}
+      title={noLiquidity ? t('You are creating a pair') : t('You will receive')}
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/AddLiquidityButton.tsx:56]: pair -> trading pair

File: apps/aptos/components/Liquidity/components/PoolPriceBar.tsx [hunk 3]
```diff
@@ -48,7 +48,7 @@ export default function PoolPriceBar({
             %
           </Text>
           <Text fontSize="14px" pt={1}>
-            {t('Share of Pool')}
+            {t('Share of Pair')}
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/PoolPriceBar.tsx:51]: Share in Trading Pair

File: apps/aptos/components/Liquidity/components/PositionCard.tsx [hunk 4]
```diff
@@ -116,7 +116,7 @@ function MinimalPositionCardView({
                 )}
                 <FixedHeightRow>
                   <Text color="textSubtle" small>
-                    {t('Share of Pool')}:
+                    {t('Share of Pair')}:
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/PositionCard.tsx:119]: Share in Trading Pair

File: apps/aptos/components/Liquidity/components/AddLiquidityButton.tsx (hunk 2 above)
Comment (Woody810) [reply apps/aptos/components/Liquidity/components/AddLiquidityButton.tsx:56]: Fixed.

File: apps/aptos/components/Liquidity/components/PoolPriceBar.tsx (hunk 3 above)
Comment (Woody810) [reply apps/aptos/components/Liquidity/components/PoolPriceBar.tsx]: Fixed.

File: apps/aptos/components/Liquidity/components/PositionCard.tsx (hunk 4 above)
Comment (Woody810) [reply apps/aptos/components/Liquidity/components/PositionCard.tsx]: Fixed.

File: apps/aptos/components/Liquidity/components/PositionCard.tsx [hunk 5]
```diff
@@ -155,7 +155,7 @@ function MinimalPositionCardView({
               🥞
             </span>{' '}
             {t(
-              "By adding liquidity you'll earn 0.17% of all trades on this pair proportional to your share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.",
+              "By adding liquidity you'll earn 0.17% of all trades on this pair proportional to your share of the pair. Fees are added to the pair, accrue in real time and can be claimed by withdrawing your liquidity.",
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/PositionCard.tsx:158]: ... share in the trading pair ...

File: apps/aptos/components/Liquidity/components/PricePoolShareSection.tsx [hunk 6]
```diff
@@ -8,7 +8,7 @@ export default function PricePoolShareSection({ farmPriceBar, noLiquidity }) {
   return (
     <LightCard padding="0px" borderRadius="20px">
       <RowBetween padding="1">
-        <Text fontSize="14px">{noLiquidity ? t('Initial prices and pool share') : t('Prices and pool share')}</Text>
+        <Text fontSize="14px">{noLiquidity ? t('Initial prices and pair share') : t('Prices and pair share')}</Text>
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/PricePoolShareSection.tsx:11]: Initial prices and share in the pair

Prices and shares of the pair

File: apps/aptos/components/Liquidity/components/common.tsx [hunk 7]
```diff
@@ -175,7 +175,7 @@ export const AddLiquidityModalHeader = ({
         </GreyCard>
       </AutoColumn>
       <RowBetween>
-        <Subtitle>{t('Your pool share')}</Subtitle>
+        <Subtitle>{t('Your pair share')}</Subtitle>
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/common.tsx:178]: Your share in the pair

File: apps/web/src/components/PositionCard/index.tsx [hunk 8]
```diff
@@ -271,7 +271,7 @@ function MinimalPositionCardView({
               </>
             ) : (
               t(
-                "By adding liquidity you'll earn 0.17% of all trades on this pair proportional to your share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.",
+                "By adding liquidity you'll earn 0.17% of all trades on this pair proportional to your share of the pair. Fees are added to the pair, accrue in real time and can be claimed by withdrawing your liquidity.",
```
Comment (chef-bun) [inline apps/web/src/components/PositionCard/index.tsx:274]: ... share in the trading pair ...

File: apps/web/src/views/AddLiquidity/AddStableLiquidity/index.tsx [hunk 9]
```diff
@@ -325,7 +325,7 @@ export default function AddStableLiquidity({ currencyA, currencyB }) {
                   <LightCard padding="0px" borderRadius="20px">
                     <RowBetween padding="1rem">
                       <Text fontSize="14px">
-                        {noLiquidity ? t('Initial prices and pool share') : t('Prices and pool share')}
+                        {noLiquidity ? t('Initial prices and pair share') : t('Prices and pair share')}
```
Comment (chef-bun) [inline apps/web/src/views/AddLiquidity/AddStableLiquidity/index.tsx:328]: Initial prices and share in the pair

Prices and shares of the pair

File: apps/web/src/views/AddLiquidity/index.tsx [hunk 10]
```diff
@@ -783,7 +783,7 @@ export default function AddLiquidity({ currencyA, currencyB }) {
                     <LightCard padding="0px" borderRadius="20px">
                       <RowBetween padding="1rem">
                         <Text fontSize="14px">
-                          {noLiquidity ? t('Initial prices and pool share') : t('Prices and pool share')}
+                          {noLiquidity ? t('Initial prices and pair share') : t('Prices and pair share')}
```
Comment (chef-bun) [inline apps/web/src/views/AddLiquidity/index.tsx:786]: Initial prices and share in the pair

Prices and shares of the pair

File: apps/aptos/components/Liquidity/components/PricePoolShareSection.tsx [hunk 11]
```diff
@@ -8,7 +8,9 @@ export default function PricePoolShareSection({ farmPriceBar, noLiquidity }) {
   return (
     <LightCard padding="0px" borderRadius="20px">
       <RowBetween padding="1">
-        <Text fontSize="14px">{noLiquidity ? t('Initial prices and pool share') : t('Prices and pool share')}</Text>
+        <Text fontSize="14px">
+          {noLiquidity ? t('Initial prices and share in the pair') : t('Prices and shares of the pair')}
```
Comment (chef-bun) [inline apps/aptos/components/Liquidity/components/PricePoolShareSection.tsx:12]: Prices and Share

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**